src/components/about-us/flippingCard.jsx

```diff
@@ -0,0 +1,28 @@
+import { useTranslations } from 'next-intl';
+import React from 'react';
+import styles from "@/styles/components/aboutus/flippingCard.module.scss";
+export async function getStaticProps({ locale }) {
+    return {
+        props: {
+            messages: (await import(`@/messages/${locale}.json`)).default
+        }
+    };
+}
+
+export default function PopupCard({ content, image, title, alt }) {
```

Comment:
change the name of this component to `FlippingCard` and make sure it has the same name in the index.jsx for the about page